OrbitGl/Batcher.cpp

```diff
@@ -103,6 +103,25 @@ void Batcher::AddTriangle(const Triangle& triangle, const Color& color, const Co
   user_data_.push_back(std::move(user_data));
 }
 
+void Batcher::AddCircle(Vec2 position, float radius, float z, Color color) {
+  std::vector<Vec2> circle_points_scaled_by_radius;
+  for (auto& point : circle_points) {
+    circle_points_scaled_by_radius.emplace_back(radius * point);
+  }
+
+  Vec2 prev_point(position[0], position[1] - radius);
+  Vec3 point_0 = Vec3(position[0], position[1], z);
+  for (size_t i = 0; i < circle_points_scaled_by_radius.size(); ++i) {
+    Vec2 new_point(position[0] + circle_points_scaled_by_radius[i][0],
```

Comment:
nit. same as above, we could use a Vec3.

Reply:
I will update the code! Thanks a lot!